.github/workflows/enteprise.yml

```diff
@@ -133,7 +135,9 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.aws_access_key_id }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.aws_secret_access_key }}
           CLONE_GITHUB_TOKEN: ${{ secrets.BOT_GITHUB_TOKEN }}
-        run: ./tests/molecule_it/script.sh verify
+        run: |
+         pipenv run molecule
+         ./tests/molecule_it/script.sh verify
```

Comment:
is this a leftover? revert it

Reply:
I do believe it is necessary, because after deleting pipenv run molecule, it stopped working

Reply:
trust the build output, it's really just printing the command help 😆 
https://github.com/Alfresco/alfresco-ansible-deployment/runs/7746557184?check_suite_focus=true#step:11:27